python-frontend/src/main/java/org/sonar/python/semantic/v2/TypeInferenceV2.java

```diff
@@ -92,7 +96,7 @@ private void inferTypesAndMemberAccessSymbols(FunctionDef functionDef) {
       .map(Parameter::name)
       .collect(Collectors.toSet());
     Set<SymbolV2> localVariables = symbolTable.getSymbolsByRootTree(functionDef);
-    inferTypesAndMemberAccessSymbols(
+    typesBySymbol = inferTypesAndMemberAccessSymbols(
```

Comment:
This is not the right place to set `typeBySymbol` for what we want to do.
This method exists for `FileInput` and for `FunctionDef`. To simulate an import, we need to consider the module-level type inference only (the `FileInput` method), as this one will override `typesBySymbol` for each function.